docs/assemblies/common_configurations.adoc

```diff
@@ -52,6 +52,27 @@ https://access.redhat.com/documentation/en-us/red_hat_satellite/6.13/html-single
 If not using Satellite version 6.13, then refer to the specific version of the
 documentation for the version of Satellite that is in use.
 
+=== Using `edpm_bootstrap_command` for configuring system proxy settings
+
+`edpm_bootstrap_command` can be used to configure system proxy settings by
+creating a file under `/etc/profile.d` that defines the proxy environment
+variables.
+
+----
+edpm_bootstrap_command: |
+    cat >/etc/profile.d/proxy.sh <<EOF
+    export http_proxy="http://USERNAME:PASSWORD@proxy-server.example.com:3128/"
+    export https_proxy="http://USERNAME:PASSWORD@proxy-server.example.com:3128/"
+    export no_proxy="localhost,127.0.0.1,::1" # Add any hosts to bypass the proxy
+    EOF
+----
```

Comment:
Should we not drop this option and text completely as it can be confusing for the user or atleast detail what all situations it won't work. The statement below (`won't always apply to tasks executed by ansible`) sounds vague to me. I think instead of /etc/profile.d/proxy.sh using environment key value pairs in `/etc/environment` would always work and we can document that.

Reply:
As discussed in slack I'll merge it and we revisit it later.